src/main/kotlin/no/nav/familie/ef/sak/behandlingsflyt/task/OpprettOppgaveTask.kt

```diff
@@ -34,12 +39,24 @@ class OpprettOppgaveTask(private val oppgaveService: OppgaveService) : AsyncTask
 
     override fun doTask(task: Task) {
         val data = objectMapper.readValue<OpprettOppgaveTaskData>(task.payload)
+        val oppgavetype = data.oppgavetype
+
+        if (oppgavetype == Oppgavetype.BehandleSak) {
+            val behandling = behandlingService.hentBehandling(data.behandlingId)
+
+            if (listOf(BehandlingStatus.FATTER_VEDTAK, BehandlingStatus.IVERKSETTER_VEDTAK, BehandlingStatus.FERDIGSTILT).contains(behandling.status)) {
```

Comment:
Kan man bruke `behandlingErLåstForVidereRedigering` som ligger i `Behandling.kt`?

Reply:
Ahaa genialt!

Oppsummering av samtale med Charlie: 
`behandlingErLåstForVidereRedigering` inneholder også `SATT_PÅ_VENT` som kunne lagd krøll, men det vil uansett bli krøll om man prøver å sette på vent rett etter angring. Fordi man må hente oppgaven for å vise feltene og om oppgaven ikke er blir det kaos🤹‍♀️